test/res/pixometer_mixed_sources.csv

```diff
@@ -1,5 +1,6 @@
 Standort,Sparte,Zählernummer,Zählerstand,Zählerstand (NT),Ablesedatum,Notizen,Notizen
 Hintereingang,Gas,28967801,46139.6,0,30.11.2021 16:31,,
+Hintereingang2,Gas2,28967801,46139.68,06,30.11.2021 16:32,,
```

Comment:
```suggestion
```

Not sure why this line was added. Having two Standorte with the same Zählernummer is ambiguous. So i would rather not have it.